.github/workflows/test.yaml

```diff
@@ -14,17 +14,35 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - uses: dtolnay/rust-toolchain@stable
+      - name: Extend space
+        uses: ./.github/actions/extend-space
```

Comment:
Is there a reason for checking this in? I've been able to use a public one with reasonable success elsewhere

Reply:
Yes, no need for this, replaced with a public one